### DISCUSSION TITLE
[建议]希望提供Windows平台删除任务计划的方法

### DISCUSSION THREAD

**Discussion by imfrea** [NewFuture/DDNS#250] [Show and tell]
如题 虽然提供了create-task.bat用于创建定期更新的任务计划 但并没有提供删除的方法或相关说明
如使用`schtasks /Delete /TN DDNS /F`命令 或手动浏览到“管理工具”->“任务计划程序”以删除

**Comment by NewFuture** [comment]
嗯，这个可以考虑添加的readme中
